src/org/scratchOrgInfoApi.ts

```diff
@@ -96,8 +95,8 @@ const buildOAuth2Options = async (options: {
 
   if (isJwtFlow && !process.env.SFDX_CLIENT_SECRET) {
     oauth2Options.privateKeyFile = options.hubOrg.getConnection().getAuthInfoFields().privateKey;
-    const retries = options?.retry || env.getNumber('SFDX_JWT_AUTH_RETRY_ATTEMPTS') || 0;
-    const timeoutInSeconds = env.getNumber('SFDX_JWT_AUTH_RETRY_TIMEOUT') || 300;
+    const retries = options?.retry ?? env.getNumber('SFDX_JWT_AUTH_RETRY_ATTEMPTS') ?? 0;
```

Comment:
nullish makes a different here--`0` on the `retry` or in the env would have been "falsy" and then went to the next fallback in line.  

I'm *assuming* that wasn't the intended behavior and there may have been some bugs eventually